api/seeds/init.ts

```diff
@@ -0,0 +1,84 @@
+import bcrypt from 'bcryptjs'
+import { USER_PROFILE } from '@locokit/definitions'
+import { Knex } from 'knex'
+
+export async function seed(knex: Knex): Promise<any> {
+  const hashPassword = await bcrypt.hash('locokit', 10)
+
+  const trx = await knex.transaction()
+
+  /**
+   * User creation
+   */
+  const usersToInsert = [
+    {
+      firstname: 'Master',
```

Comment:
Hmmm nope

Reply:
Done

Reply:
Nope it still there

Reply:
No.
Updated FirstName and lastName was for this comment https://github.com/locokit/locokit/pull/218#discussion_r1190071778

The issue here is the "name" chosen… it is annoying

Reply:
All right, I update the name to John. 
Please be more precise in your comments, I didn't originally understand what was exactly the point.

Reply:
It would have been nice to change the surname too…